dts/ulticontroller4.0.dtsi

```diff
@@ -19,4 +19,37 @@
 		#gpio-cells = <2>;
 		reg = <0x24>;
 	};
+
+	pca9632: pca9632 {
+        compatible = "nxp,pca9632";
+        #address-cells = <1>;
+        #size-cells = <0>;
+        reg = <0x60>;
+        nxp,totem-pole;
+        nxp,hw-blink;
```

Comment:
Is the indentation level here supposed to be different from the rest of the file?